requirements.txt

```diff
@@ -4,10 +4,11 @@ django-ratelimit==1.0.0
 django-wizard-builder==0.0.3
 
 PyNaCl==1.0.1
-python-gnupg==0.3.8
+gnupg==2.0.2
```

Comment:
i'm a little wary of this bc this code change is a high risk for data loss and gpg is so unfortunately fiddly. would like to try it with python-gnupg on Travis once, at least. (as a side note, these two packages need to get it together...)